### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by manzitresor**
### Implemented Feature

- [ ] Add any missing JSX elements to your project (without adding extra functionality)
- [ ] Your final design should match Zeplin's design (this includes all elements + the same styling)
- [ ] You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -9,13 +11,21 @@ const navLinks = [
 function Navbar() {
   return (
     <>
-      {
+      <div className="navbar-container">
```
Comment (Lucas-Masaba) [inline src/components/Navbar.js:14]: - [ ] The bottom border should be darker as seen in the original design. 
### Your design
image
### Required Design
image

File: src/components/Completed.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
+
+function Completed() {
+  return (
+    <>
+      <div className="main-completed">
```
Comment (Lucas-Masaba) [inline src/components/Completed.js:8]: - [ ] Please bring progress container closer to the chapter container as seen in the original design.
### Your Design
image
### Original design
image

File: src/components/AddBook.js [hunk 3]
```diff
@@ -20,15 +20,20 @@ function AddBook() {
...
   };
 
   return (
     <>
-      <form onSubmit={handleSubmit}>
-        <input type="text" value={title} placeholder="Book Title" onChange={handletitleChange} />
-        <input type="text" value={author} placeholder="author" onChange={handleSelectChange} />
-        <AddBookBtn />
-      </form>
+      <div className="add-container">
```
Comment (Lucas-Masaba) [inline src/components/AddBook.js:29]: - [ ] Please make sure that this form has the same width as that in the book cards above.
### Your implementation
image
### Required
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @manzitresor  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Descriptive pull request message :writing_hand: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by manzitresor** [comment]
hello@Lucas-Masaba 
thanks for review feed back ,gona work on it 🤝

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @manzitresor 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Use GitHub Flow
- ✔️ Add styling

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by manzitresor** [comment]
Hello @abdulhamiid 
Thank you so much for approving this project.

**Pull request merged**